src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,130 @@
+package duke;
+
+import java.util.Scanner;
+
+public class Duke {
+
+    static final Task[] TASKS = new Task[101];
+    static int tasksI = 1;
+
+    public static void main(String[] args) {
+        // print intro message
+        start();
+        // run bot (decode task)
+        run();
+    }
+
+    public static void start() {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+    }
+
+    public static void list() {
+        System.out.println("Here are the tasks in your list: ");
+        for (int i = 1; i < tasksI; i++) {
+            Task currTask = TASKS[i];
+            System.out.println(i + ". " + currTask.toString());
+        }
+    }
+
+    public static void mark(String word, Task task) {
+        task.mark();
+        System.out.println("Nice! I've marked this task as done: ");
+        System.out.println(task.toString());
+    }
+
+    public static void unmark(String word, Task task) {
+        task.unmark();
+        System.out.println("OK, I've marked this task as not done yet: ");
+        System.out.println(task.toString());
+    }
+
+    public static void deadline(String taskDescript) {
+        int bySize = 3;
+        String by = taskDescript.substring(taskDescript.indexOf("by") + bySize);
+        Deadline deadline = new Deadline(taskDescript, by);
+        TASKS[tasksI] = deadline;
+        System.out.println("Got it. I've added this task: ");
+        System.out.println(deadline.toString());
```

Comment:
These 2 lines seems to behave similarly when compared to line 65-66 and 78-79. Perhaps you can consider extracting them to consolidate the printing of such methods that can be easily refactored in future commits if required